code-samples/popular-topics/permissions/13/index.js

```diff
@@ -11,22 +11,22 @@ client.on('interactionCreate', interaction => {
 	if (!interaction.isCommand()) return;
 	if (!interaction.channel.permissionsFor(client.user).has('SEND_MESSAGES')) return;
 
-	const botPerms = ['MANAGE_MESSAGES', 'KICK_MEMBERS', 'MANAGE_ROLES', 'MANAGE_CHANNELS'];
+	const botPerms = [Permissions.FLAGS.MANAGE_MESSAGES, Permissions.FLAGS.KICK_MEMBERS, Permissions.FLAGS.MANAGE_ROLES, Permissions.FLAGS.MANAGE_CHANNELS];
```

Comment:
```suggestion
	const botPerms = [
		Permissions.FLAGS.MANAGE_MESSAGES,
		Permissions.FLAGS.KICK_MEMBERS,
		Permissions.FLAGS.MANAGE_ROLES,
		Permissions.FLAGS.MANAGE_CHANNELS,
	];
```
Lint because it becomes long for a single line